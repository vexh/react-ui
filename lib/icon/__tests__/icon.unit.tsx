import React from 'react';
import renderer from 'react-test-renderer';
import Icon from '../icon';

test('Icon 是个 svg', () => {
    const component = renderer.create(
        <Icon name="wechat" />,
    );

    expect(component).toMatchSnapshot();
})