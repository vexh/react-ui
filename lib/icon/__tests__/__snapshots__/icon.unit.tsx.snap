// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon 是个 svg 1`] = `
<svg
  className="react-ui"
>
  <use
    xlinkHref="#wechat"
  />
</svg>
`;
